ush/config_defaults.sh

```diff
@@ -431,6 +460,7 @@ FCST_LEN_HRS="24"
 EXTRN_MDL_NAME_ICS="FV3GFS"
 EXTRN_MDL_NAME_LBCS="FV3GFS"
 LBC_SPEC_INTVL_HRS="6"
+EXTRN_MDL_LBCS_OFFSET_HRS="3"
```

Comment:
As per the comment in the set_extrn_mdl_params.sh file, this could be left blank by default to retain previous behavior.

Reply:
Done.